client/templates/home/home_logged_out.html

```diff
@@ -32,8 +32,8 @@
 
           <div id="intro">
             <h2>Learn and chat about code!</h2>
-            <p class="intro">CodeBuddies is a community of 4000+ code learners who help each other through conversations on <strong>Slack</strong> and peer-to-peer organized <strong>study groups</strong> and <strong>virtual hangouts</strong>.</p>
-            <p>This project is free and <a href="http://opencollective.com/codebuddies" target="_blank" class="link">transparent on Open Collective</a>, non-profit, <a href="https://github.com/codebuddies/codebuddies" target="_blank" class="link">open-sourced</a>, and powered by a fantastic team of core volunteer <a href="/about" class="link">contributors</a>.</p>
+            <p class="intro">CodeBuddies is a global community of code learners who help each other through conversations on <strong>Slack</strong> and peer-to-peer organized <strong>study groups</strong> and <strong>virtual hangouts</strong>.</p>
+            <p>This project is free and <a href="http://opencollective.com/codebuddies" target="_blank" class="link">transparent on Open Collective</a>, non-for-profit, <a href="https://github.com/codebuddies/codebuddies" target="_blank" class="link">open-sourced</a>, and powered by a fantastic team of core volunteer <a href="/about" class="link">contributors</a>.</p>
```

Comment:
@wrejcodes Slight correction here. It should say "not-for-profit", not "non-for-profit".

Reply:
oops sorry! Will fix real quick

Reply:
@wuworkshop should be resolved now!